core/templates/components/oppia-angular-root.component.ts

```diff
@@ -198,7 +199,14 @@ export class OppiaAngularRootComponent implements AfterViewInit {
     OppiaAngularRootComponent.rteElementsAreInitialized = true;
   }
 
+  getContextServiceForTesting(): ContextService {
+    return this.contextService;
+  }
```

Comment:
Hi @mon4our, we should not be adding any functions to a component for solely testing -- do we need this?

Reply:
I used it as eslint tests were failing due to contextService being private. Removed the function and disabled eslint for the problematic line. Done!